src/snoowrap.js

```diff
@@ -233,6 +233,32 @@ const snoowrap = class snoowrap {
       return requester;
     });
   }
+  static fromInstalledClientAuth ({
+    userAgent = isBrowser ? global.navigator.userAgent : requiredArg('userAgent'),
+    clientId = requiredArg('clientId'),
+    deviceId = requiredArg('deviceId'),
+    endpointDomain = 'reddit.com'
+  }) {
+    return this.prototype.credentialedClientRequest.call({
+      clientId,
+      deviceId,
```

Comment:
Does `deviceId` actually do anything here? I don't think `credentialedClientRequest` accesses any `deviceId` property.

Reply:
Ah, I think i miss understood something when copying from the other auth call. I'll remove it.